.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Issue
```

Comment:
There should ideally be a section to specify the version on which the bug was observed.